scipy/stats/_continuous_distns.py

```diff
@@ -55,15 +57,26 @@ def _remove_optimizer_parameters(kwds):
 
 
 def _call_super_mom(fun):
-    # if fit method is overridden only for MLE and doesn't specify what to do
-    # if method == 'mm', this decorator calls generic implementation
+    # If fit method is overridden only for MLE and doesn't specify what to do
+    # if method == 'mm' or with censored data, this decorator calls the generic
+    # implementation.
     @wraps(fun)
-    def wrapper(self, *args, **kwds):
+    def wrapper(self, data, *args, **kwds):
         method = kwds.get('method', 'mle').lower()
-        if method != 'mle':
-            return super(type(self), self).fit(*args, **kwds)
+        censored = isinstance(data, CensoredData)
+        if method == 'mm' or (censored and data.num_censored() > 0):
+            return super(type(self), self).fit(data, *args, **kwds)
         else:
-            return fun(self, *args, **kwds)
+            if censored:
+                # data is an instance of CensoredData, but actually holds
+                # no censored values, so replace it with the array of
+                # uncensored values.
+                data = data._x
+            return fun(self, data, *args, **kwds)
+
+    if hasattr(fun, '__doc__'):
+        wrapper.__doc__ = fun.__doc__
+
```

Comment:
```suggestion
```
Isn't this sort of thing taken care of by `@wraps`?

```python3
from functools import wraps

def make_wrapper(fun):
    @wraps(fun)
    def wrapper(self, *args, **kwargs):
        return fun(self, *args, **kwargs)
    return fun

class MyClass:
    @make_wrapper
    def method(self):
        """This is a docstring"""
        
o = MyClass()
print(o.method.__doc__)  # This is a docstring

wrapped = make_wrapper(o.method)
assert wrapped.__doc__ == o.method.__doc__

help(o.method)
# Help on method method in module __main__:

# method() method of __main__.MyClass instance
#     This is a docstring
```

Reply:
Yup, fixed in https://github.com/scipy/scipy/pull/13699/commits/b60247458057a7368be8cd9b63b69449bdeb854e